findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Listing.java

```diff
@@ -0,0 +1,134 @@
+package com.tp.wrc.findmyparadise.models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Listing")
```

Comment:
Not necessary to explicitly name the table since it will be the same name as the Java class.